src/utils/config.ts

```diff
@@ -58,3 +58,8 @@ export const network = networks[networkId];
 export const signInContractId = networkId === 'testnet' ? 'v1.social08.testnet' : 'social.near';
 export const termsDomainName = `${process.env.NEXT_PUBLIC_TOS_SUBDOMAIN_NAME}/ipfs/${process.env.NEXT_PUBLIC_TERMS_CID}`;
 export const privacyDomainName = `${process.env.NEXT_PUBLIC_TOS_SUBDOMAIN_NAME}/ipfs/${process.env.NEXT_PUBLIC_PRIVACY_CID}`;
+export const notificationApplicationServerKey = process.env.NEXT_PUBLIC_NOTIFICATIONS_APPLICATION_SERVER_KEY;
+export const notificationsHostName = process.env.NEXT_PUBLIC_NOTIFICATIONS_HOSTNAME;
+export const notificationsGatewayUrl = process.env.NEXT_PUBLIC_NOTIFICATIONS_GATEWAY_URL ?? 'https://near.org';
```

Comment:
@milenakrawczyk is this gateway url should be the same to every environment we have or we can rely on `NEXT_PUBLIC_HOSTNAME` which will be different for each environment?

Reply:
That's why I asked you about it. I'll leave it as is. Thanks for reply!

Reply:
FYI: all this key-values you can find in 1Password and I've added them to Vercel

cc @milenakrawczyk @calebjacob